flink-ml-lib/src/main/java/org/apache/flink/ml/common/param/HasFeatureColsDefaultAsNull.java

```diff
@@ -0,0 +1,28 @@
+package org.apache.flink.ml.common.param;
+
+import org.apache.flink.ml.param.Param;
+import org.apache.flink.ml.param.ParamValidators;
+import org.apache.flink.ml.param.WithParams;
+
+/** Params of the names of the feature columns used for training in the input table. */
+public interface HasFeatureColsDefaultAsNull<T> extends WithParams<T> {
+    /**
+     * @cn-name 特征列名数组
```

Comment:
Can we make the Java doc all in English?

Also what is the different between `HasFeatureColsDefaultAsNull` and `HasFeatureCols`?

Reply:
There seems to have been an agreement in Lindong's review that we will only have `HasFeatureCols`, and no more `HasFeatureColsDefaultAsNull`.

Reply:
OK, I will refine it.

Reply:
done